src/App.tsx

```diff
@@ -1,61 +1,34 @@
+import React, { useState } from 'react';
+
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Todo } from './types';
+import { TodoForm } from './components/TodoForm/TodoForm';
+import { getUserById } from './utils/getUserById';
+
+export const App: React.FC = () => {
+  const preparedTodos: Todo[] = todosFromServer.map(todo => {
+    return {
+      ...todo,
+      user: getUserById(todo.userId) || null,
+    };
```

Comment:
You're preparing your data correctly in the App component (checklist item 5), which is a good practice. However, you're importing and using the `getUserById` function directly in this component. According to the checklist, you should avoid using `import` in non-root components. Consider moving the `getUserById` function to the App component and passing the result down to child components as props.